### DISCUSSION TITLE
Problem to access my old github account

### DISCUSSION THREAD

**Discussion by khalilMaat** [community/community#127785] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

i have problem with my old account 

when i am trying to login it return: 

Access to your account has been suspended due to a violation of our [Terms of Service](https://docs.github.com/articles/github-terms-of-service).
Please [contact support](https://support.github.com/contact/reinstatement) for more information.

please help me i need this account 
![Screenshot 2024-06-08 081011](https://github.com/community/community/assets/152015310/f0c8547e-b236-4283-9a77-2152a858a7c6)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @khalilMaat ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

 The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

Also for future discussions: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
